src/frontend/web_application/src/services/message/index.js

```diff
@@ -9,3 +9,13 @@ export const sortMessages = (messages, reversed) => messages.sort((a, b) => {
 export const renderParticipant = participant => `${participant.label} (${participant.address})`;
 
 export const getLastMessage = messages => sortMessages(messages, true)[0];
+
+export const getAuthor = message => message.participants && message.participants.find(participant => participant.type === 'From');
+
+export const isMessageFromUser = (message, user) => {
+  const author = getAuthor(message);
+  const authorId = author && author.contact_ids[0];
```

Comment:
in case of an unexpected reason there's more than one contact_id, it may not work correctly.

Also the name of the variable is not self-explanatory:

authorId -> authorContactId
userId -> userContactId

Imo, you should use `author.contact_ids.some(......)` to compare the userContactId.